### ISSUE TITLE
Agent adds identical domain entries to glpi_domains after each inventory task

### ISSUE THREAD

**Issue by mleyh** [glpi-project/glpi-agent#424]
### Bug reporting acknowledgment

Yes, I read it

### Professional support

None

### Describe the bug

After updating to GLPI 10.0.7, the GLPI Agent no longer works as intended.
The following error message appears in the log:

```
[Tue Jun 20 13:49:29 2023][error] [http client] failed to uncompress content starting with: <br />
<b>Warning</b>: getFromDBByCrit expects to get one result, 7 found in query "SELECT `id` FROM `glpi_domains` WHERE `name` = 'domainxyz.local'". in <b>D:\xampp\htdocs\glpi\src\CommonDBTM.php</b> on line <b>397</b><br />
xœ«VJ(È,J,ÉÌÏS²R22QÒQ*.I,)-
```

Obviously, with each new inventory task, another identical domain entry is added to the glpi_domains table.

### To reproduce

1. Update to GLPI 10.0.7
2. Run an inventory task and view glpi-agent.log

### Expected behavior

No additional entry in glpi_domains table 

### Operating system

Windows

### GLPI Agent version

1.4

### GLPI version

10.0.x (See additional context below)

### GLPIInventory plugin or FusionInventory for GLPI plugin version

Not applicable

### Additional context

10.0.7

**Comment by trasher** [comment]
This does not seems an agent issue, and it's already fixed on GLPI for 10.0.8; try with nightly build.

**Comment by mleyh** [comment]
Nightly build 10.0.8 resolves this problem.
Thank you.
